lib/src/util/event_helpers.dart

```diff
@@ -14,92 +14,5 @@
-SyntheticKeyboardEvent wrapNativeKeyboardEvent(KeyboardEvent nativeKeyboardEvent) {
-  return SyntheticKeyboardEvent(
-    nativeKeyboardEvent.bubbles,
-    nativeKeyboardEvent.cancelable,
-    nativeKeyboardEvent.currentTarget,
-    nativeKeyboardEvent.defaultPrevented,
-    nativeKeyboardEvent.preventDefault,
-    nativeKeyboardEvent.stopPropagation,
-    nativeKeyboardEvent.eventPhase,
-    null, // isTrusted
-    nativeKeyboardEvent,
-    nativeKeyboardEvent.target,
-    nativeKeyboardEvent.timeStamp,
-    nativeKeyboardEvent.type,
-    nativeKeyboardEvent.altKey,
-    nativeKeyboardEvent.charCode == null ? null : String.fromCharCode(nativeKeyboardEvent.charCode),
-    nativeKeyboardEvent.charCode,
-    nativeKeyboardEvent.ctrlKey,
-    null, // locale
-    nativeKeyboardEvent.location,
-    null, // key
-    nativeKeyboardEvent.keyCode,
-    nativeKeyboardEvent.metaKey,
-    nativeKeyboardEvent.repeat,
-    nativeKeyboardEvent.shiftKey);
-}
-
-/// Helper util that wraps a native [MouseEvent] in a [SyntheticMouseEvent].
-///
-/// Used where a native [MouseEvent] is given and a [SyntheticMouseEvent] is needed.
-SyntheticMouseEvent wrapNativeMouseEvent(MouseEvent nativeMouseEvent) {
-  return SyntheticMouseEvent(
-    nativeMouseEvent.bubbles,
-    nativeMouseEvent.cancelable,
-    nativeMouseEvent.currentTarget,
-    nativeMouseEvent.defaultPrevented,
-    nativeMouseEvent.preventDefault,
-    nativeMouseEvent.stopPropagation,
-    nativeMouseEvent.eventPhase,
-    null, // isTrusted
-    nativeMouseEvent,
-    nativeMouseEvent.target,
-    nativeMouseEvent.timeStamp,
-    nativeMouseEvent.type,
-    nativeMouseEvent.altKey,
-    nativeMouseEvent.button,
-    null, // buttons
-    null, // clientX
-    null, // clientY
-
-    nativeMouseEvent.ctrlKey,
-    null, // dataTransfer
-    nativeMouseEvent.metaKey,
-    null, // pageX
-    null, // pageY
-    nativeMouseEvent.relatedTarget,
-    null, // screenX
-    null, // screenY
-    nativeMouseEvent.shiftKey);
-}
-
-/// If the consumer specifies a callback like `onChange` on one of our custom form components that are not *actually*
-/// form elements - we still need a valid [SyntheticFormEvent] to pass as the expected parameter to that callback.
-///
-/// This helper method generates a "fake" [SyntheticFormEvent], with nothing but the `target` set to [element],
-/// `type` set to [type] and `timeStamp` set to the current time. All other arguments are `noop`, `false` or `null`.
-SyntheticFormEvent fakeSyntheticFormEvent(Element element, String type) {
-  return SyntheticFormEvent(
-    false,
-    false,
-    element,
-    false,
-    () {},
-    () {},
-    Event.AT_TARGET,
-    false,
-    null,
-    element,
-    DateTime.now().millisecondsSinceEpoch,
-    type
-  );
-}
+export 'package:react/react.dart'
+    show wrapNativeKeyboardEvent, wrapNativeMouseEvent, fakeSyntheticFormEvent, SyntheticEventTypeHelpers, DataTransferHelper;
```

Comment:
#nit SyntheticEventTypeHelpers an DataTransferHelper are being exported unnecessarily

Reply:
Ahh yep, good catch. I realized that all of that really should have just been exported with the event stuff from react-dart anyway, so I just move all the exports into `over_react.dart`. Let me know if I missed something though and it was important to keep this file!